README.md

```diff
@@ -29,3 +29,31 @@ Visit our **readthedocs** page for other setup and configuration at:
 <https://treeherder.readthedocs.io>
 
 File any bugs you may encounter [here](https://bugzilla.mozilla.org/enter_bug.cgi?product=Tree+Management&component=Treeherder).
+
+
+#### Contributing
+
+Firstly, ask if the bug is available and manifest your interest in contributing to it. One of the maintainers will **assign** you the bug. Only then you should work on it.
+
+After adressing the issue, make sure [every test passes](https://treeherder.readthedocs.io/testing.html) before sending a pull request.
+
+We also recommend using [`git rebase`](https://git-scm.com/docs/git-rebase). There are less conflicts this way and the git history is cleaner.
```

Comment:
Yes, this is good to mention. "We also recommend setting an "upstream" remote that points to the github repo, in addition to "origin" that points to your fork. You should then frequently use `git rebase upstream` rather than merging from your fork to keep your branch current. There are less conflicts this way and the git history is cleaner."